src/light_client.rs

```diff
@@ -283,84 +273,45 @@ pub async fn process_block(
 					.collect::<Vec<_>>();
 				rpc_fetched.extend(partition_fetched_filtered.clone());
 			}
+			let rpc_call_duration = begin.elapsed();
```

Comment:
Wouldn't this always measure aggregated time from the begining of the loop?

Reply:
Yeah, the idea is to measure total time needed to retrieve all of the data from the node.